### DISCUSSION TITLE
Help needed on Showing the Fan/Power LCD numbers

### DISCUSSION THREAD

**Discussion by Diomioleo** [artisan-roaster-scope/artisan#1234] [Q&A] [answered]
![image](https://github.com/artisan-roaster-scope/artisan/assets/93377102/fb163a64-39ae-4da8-b45e-0753e1a941f7)

I am using a Santoker Q5 air roaster，using the default extra device setting from the 2.8.4 Ver. however when I tick the LCD1 LCD2, it shows Fan and Power LCD, but it doesnt tell the real time number of the Fan and  Power, like below:

![image](https://github.com/artisan-roaster-scope/artisan/assets/93377102/5a536428-9308-4314-95d9-bb247765e784)

It can control the Fan/Power from the sliders Air/Burner, Is there any other settings should I do?

Thanks.

**Comment by MAKOMO** [comment]
Could be that the Q10 (opposed to the Q5) is not sending this data out. I have no machine to test. Could you please attach your Artisan log file here by drag-and-drop the `artisan.log` file (export via renamed to end in `.txt` here on a Github comment box of this thread. You can retrieve your log file by a click on the plus icon in the upper left corner of the Artisan main window while holding the ALT key (Windows) or the OPTION key (macOS). An email should open with the `artisan.log` file attached. I will have a look. Thanks for your support!

**Reply by MAKOMO** [reply]
Oh, and if you could attach additionally a recorded Artisan profile as .alog file (again you have to rename it to end in `.txt` to be able to attach it here per drag-and-drop) that would help as well. Thanks!

**Reply by Diomioleo** [reply]
Thank you! For helping here, please find log file below, I will share the profile after I make one, thanks.
[artisan.log.txt](https://github.com/artisan-roaster-scope/artisan/files/12092445/artisan.log.txt)

**Reply by MAKOMO** [reply]
Could you please also attach one of your profiles recorded on the Q10 as well as your Artisan settings used for this recording (export the Artisan settings ´artisan-settings.aset` via menu `Help >> Save Settings` and rename to `artisan-settings.aset.txt` to be able to drag-and-drop them here on a text field on Github). Thanks!

**Reply by Diomioleo** [reply]
[23-07-19_1702.alog.txt](https://github.com/artisan-roaster-scope/artisan/files/12093268/23-07-19_1702.alog.txt)
[artisan.log.txt](https://github.com/artisan-roaster-scope/artisan/files/12093272/artisan.log.txt)


Here comes the other 2files，please check，thanks a lot in advance！

**Reply by MAKOMO** [reply]
Thanks for the profile. However, you attached the log file again instead of the Artisan settings. Could you please attach those settings as well. Thanks!

**Comment by Diomioleo** [comment]
[artisan-settings.aset.txt](https://github.com/artisan-roaster-scope/artisan/files/12093939/artisan-settings.aset.txt)
My Bad...by typo... please find this one!

**Reply by MAKOMO** [reply]
Thanks!

**Comment by MAKOMO** [comment, marked as answer]
Seems that your Q10 is just not sending that data out, at least not in a way as specified by the specified protocol. Sorry. Maybe you can contact Santoker and ask for clarification.

**Reply by Diomioleo** [reply]
Mako, Thanks for you time! Very appreciated.
